cluster/src/main/java/org/apache/iotdb/cluster/server/handlers/caller/CheckStatusHandler.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.iotdb.cluster.server.handlers.caller;
+
+import org.apache.iotdb.cluster.rpc.thrift.CheckStatusResponse;
+import org.apache.thrift.async.AsyncMethodCallback;
+
+public class CheckStatusHandler implements AsyncMethodCallback<CheckStatusResponse> {
+
+  CheckStatusResponse checkStatusResponse;
+
+  public CheckStatusHandler() {
+    this.checkStatusResponse = new CheckStatusResponse();
```

Comment:
Is it necessary to `new` here? I think the handler should always have the result set by `onComplete`.

Reply:
I used GenericHandler instead in the updated version. This class is removed.